lua/telescope/pickers.lua

```diff
@@ -422,9 +423,12 @@ function Picker:find()
       PERF("Displayed FN Amount", displayed_fn_amount)
     end)
 
+
+    local start = vim.fn.reltimefloat(vim.fn.reltime())
```

Comment:
let's remove these.

This can be done either by profiling later or by adding it in the finder directly.